cssom-view-1/Overview.bs

```diff
@@ -359,13 +359,53 @@ the following steps must be run:
   </ul>
   ...then perform a <a>smooth scroll</a> of <var>box</var> to <var>position</var>. Once the position has finished updating, emit the <a event>scrollend</a> event.
   Otherwise, perform an <a>instant scroll</a> of <var>box</var> to <var>position</var>. After an <a>instant scroll</a> emit the <a event>scrollend</a> event.
-  
-  Note: If the scroll position did not change as a result of the user interaction or programmatic invocation, where no translations were applied as a result, then no <a event>scrollend</a> event fires because no scrolling occured.
 
-  Note: In the case the VisualViewport was scrolled, yielding no translation changes in the document, emit the <a event>scrollend</a> on the VisualViewport just as the <a event>scroll</a> event is emitted for VisualViewport scrolling.
+  Note: If the scroll position did not change as a result of the user interaction or programmatic invocation, where no translations were applied as a result, then no <a event>scrollend</a> event fires because no scrolling occured.
 </ol>
 
-Scroll is <dfn lt="scroll completed">completed</dfn> when the scroll position has no more pending updates or translations and the user has completed their gesture. Scroll position updates include smooth or instant mouse wheel scrolling, keyboard scrolling, scroll-snap events, or other APIs and gestures which cause the scroll position to update and possibly interpolate. User gestures like touch panning or trackpad scrolling aren't complete until pointers or keys have released. 
+When a user agent is to <dfn for="viewport" export>perform a scroll</dfn> of a <a>viewport</a> to a given position <var>position</var> and optionally a scroll behavior <var>behavior</var>
+(which is "<code>auto</code>" if omitted) it must perform a coordinated viewport scroll by following these steps:
+
+1. Let <var>doc</var> be the associated document of the <a>viewport</a>
```

Comment:
Other similar steps that do this use something like:

```
Let <var>doc</var> be the <a>viewport's</a> associated {{Document}}
```

Is there a reason to write it this way for this step?

Reply:
Nope, just missed it. Fixed in latest commit.